.github/workflows/check_tox.yml

```diff
@@ -0,0 +1,30 @@
+name: tox
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 3
+      matrix:
+        python-version: [3.5, 3.6, 3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python3 -m venv venv
+        . venv/bin/activate
+        python -m pip install --upgrade pip setuptools
+        pip install tox tox-gh-actions '.[dev]'
+    - name: Test with tox
+      run: |
+        . venv/bin/activate
+        tox
```

Comment:
here you are running docs and lint for each python version. whereas before it was only done for python 3.7

You should remove the two tox -e here and edit tox.ini as following: `py3.7: py37,docs,lint`